src/test/java/CurrentChargingRangeCountTest.java

```diff
@@ -0,0 +1,41 @@
+import static org.junit.Assert.assertEquals;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.Map;
+
+import org.junit.Test;
+
+public class CurrentChargingRangeCountTest {
+
+	CurrentChargingRangeCount chargingCount = new CurrentChargingRangeCount();
+
+
+	@Test
+	public void test_invalid_range() throws Exception {
```

Comment:
the need is to find continuous ranges within a given input and then print the number of elements within each range.
But since you have implemented in a different way, the comment for this implementation is as follows,

input data should not be hardcoded.
input data should contain a variations with only one element also

Reply:
is this comment addressed?